saleor/tests/fixtures.py

```diff
@@ -495,6 +495,27 @@ def shipping_zone(db):  # pylint: disable=W0613
     return shipping_zone
 
 
+@pytest.fixture
+def shipping_zones(db):
```

Comment:
Maby we can use `bulk_create`?

Reply:
Should I update every fixture then? I was basing on existing ones.

Reply:
Done